src/Api/Api.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <UserSecretsId>passwordless-api</UserSecretsId>
+      <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
```

Comment:
What is this used for?

Reply:
Ah forgot about that. Good catch. This was auto-generated when I added docker support for API. I can roll those back.